src/ngx_http_lua_subrequest.c

```diff
@@ -382,15 +404,62 @@ ngx_http_lua_ngx_location_capture_multi(lua_State *L)
 
             type = lua_type(L, -1);
 
-            if (type == LUA_TNIL) {
-                method = NGX_HTTP_GET;
+            switch (type) {
+                case LUA_TNIL:
```

Comment:
Style: we need to align each `case` statement with the column of their `switch`. That is

```
switch (type) {
case LUA_TNIL:
    ...
case ...
    ...
}
```